ledger/src/blockstore_processor.rs

```diff
@@ -320,7 +320,6 @@ pub struct ProcessOptions {
     pub new_hard_forks: Option<Vec<Slot>>,
     pub frozen_accounts: Vec<Pubkey>,
     pub debug_keys: Option<Arc<HashSet<Pubkey>>>,
-    pub additional_builtins: Option<Builtins>,
```

Comment:
Removed `additional_builtins` from the `ProcessOptions`. Because this isn't rather _optional_. Now, the `solana-ledger` crate is responsible for populating this.